help_to_heat/portal/download_views.py

```diff
@@ -109,3 +151,18 @@ def create_referral_csv(referrals, file_name):
     for row in rows:
         writer.writerow(row)
     return response
+
+
+def create_feedback_csv(feedbacks, file_name):
```

Comment:
Probably worth us extracting a common helper for CSV download, to avoid having to explicitly add fixes (like the BOM_UTF8 stuff) to multiple places.